set.mm

```diff
@@ -520225,6 +520231,62 @@ _modus ponens_ (our ~ ax-mp ), but he later qualifies this.
 
 $)
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+                Formally define terms such as Reflexivity
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+
+EXPERIMENTAL.
+Several terms are used in comments but not directly defined in set.mm.
+For example, there are proofs that a number of
+specific relationships are reflexive, but there is no formal definition
+of what being reflexive actually *means*.
+Stating the relationships directly, instead of defining a broader test
+such as being reflexive, can reduce proof size (because the definition
+of does not need to be expanded later).
+A disadvantage, however, is that there are several terms that are
+widely used in comments but do not have a clear formal definition.
+
+Here we define wffs that formally define some of these key terms.
+The intent isn't to use these directly, but to instead provide a clear
+formal definition of widely-used mathematical terminology
+(we even use this terminology within the comments of set.mm itself).
+
+We could define these using extensible structures, but these definitions
+don't really require the use of extensibel structures. Thus, to make it
+clearer that these can apply outside of extensible structures, we do not use
+extensible structures here.
+$)
+
+  $c Reflexive $. $( True iff a relation is reflexive. $)
+
+  $( Extend wff definition to include "Reflexive" applied to a class, which is
+     true iff class R is a reflexive relationship over the set A. See
+     ~ df-reflexive .  (Contributed by David A. Wheeler, 1-Dec-2019.) $)
+  wreflexive $a wff Reflexive A R $.
+
+  ${
+    $d A R x $.
+    $( Define relexive relationship; R is reflexive over the set A iff
+       ` A. x e. A x R x ` .  (Contributed by David A. Wheeler, 1-Dec-2019.) $)
+    df-reflexive $a |- ( Reflexive A R <-> ( A. x e. A x R x ) ) $.
+  $}
+
+  $c Irreflexive $. $( True iff a relation is irreflexive. $)
+
+  $( Extend wff definition to include "Irreflexive" applied to a class, which
+     is true iff class R is an irreflexive relationship over the set A. See
+     ~ df-irreflexive .  (Contributed by David A. Wheeler, 1-Dec-2019.) $)
+  wirreflexive $a wff Irreflexive A R $.
+
+  ${
+    $d A R x $.
+    $( Define irrelexive relationship; R is irreflexive over the set A iff
+       ` A. x e. A -. x R x ` .  Note that a relationship can be neither
+       reflexive nor irreflexive.  (Contributed by David A. Wheeler,
+       1-Dec-2019.) $)
+    df-irreflexive $a |- ( Irreflexive A R <-> ( A. x e. A -. x R x ) ) $.
```

Comment:
Same, no need for the parenthesis around ` A. x e. A ... `. set.mm likes them so much we tend to put too many!